code/builders/builder-vite/src/vite-config.ts

```diff
@@ -112,6 +113,7 @@ export async function pluginConfig(options: Options) {
       },
     },
     await externalGlobalsPlugin(externals),
+    pluginWebpackStats({ workingDir: process.cwd() }),
```

Comment:
You're loosing the ability here for users to configure this directory, I don't know how important that is.

Reply:
Tom and I talked about it.  I think it's probably fine to restrict to just cwd.  That's what webpack does.  And if users start to report issues, they can filter out this plugin and use `vite-plugin-turbosnap` as a workaround, and we can consider adding the option back in if we hear from folks that this is an issue.

Reply:
That's it. I'm not aware of any reason you'd want to change it, but open to it if needed.